text/text.go

```diff
@@ -7,171 +7,170 @@ import (
 	"time"
 
 	"github.com/ONSdigital/dp-api-clients-go/v2/dataset"
+	"github.com/ONSdigital/dp-cantabular-metadata-exporter/custom"
 )
 
-func NewMetadata(m *dataset.Metadata, isCustom bool, filterOutputID, downloadServiceURL string) []byte {
-	if isCustom {
-		var b bytes.Buffer
-		var titleDims []string
-		dt := time.Now()
-		issuedDate := dt.Format("01-02-2006")
+// New .txt metadata returned based on provided metadata
```

Comment:
Godoc line should stat with the name of the element it describes:
```suggestion
// NewMetadata returns a .txt metadata based on provided metadata
```